Modules/Files/src/Files.Infrastructure/Persistence/Database/EntityTypeConfigurations/FileMetadataEntityTypeConfiguration.cs

```diff
@@ -11,5 +11,7 @@ public void Configure(EntityTypeBuilder<FileMetadata> builder)
         builder.HasIndex(m => m.CreatedBy);
 
         builder.Property(m => m.CipherHash).IsRequired();
+
+        builder.Ignore(m => m.Content);
```

Comment:
When you modify an EntityTypeConfiguration, you should always create a migration. This just creates empty `Up` and `Down` methods, but the DbContextModelSnapshot should update.